src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutPage.cs

```diff
@@ -21,17 +22,22 @@ public record LoadoutPageContext : IPageFactoryContext
 public class LoadoutPageFactory : APageFactory<ILoadoutViewModel, LoadoutPageContext>
 {
     private readonly ISettingsManager _settingsManager;
+    private readonly IConnection _connection;
+
     public LoadoutPageFactory(IServiceProvider serviceProvider) : base(serviceProvider)
     {
         _settingsManager = serviceProvider.GetRequiredService<ISettingsManager>();
+        _connection = serviceProvider.GetRequiredService<IConnection>();
     }
 
     public static readonly PageFactoryId StaticId = PageFactoryId.From(Guid.Parse("62fda6ce-e6b7-45d6-936f-a8f325bfc644"));
     public override PageFactoryId Id => StaticId;
 
     public override ILoadoutViewModel CreateViewModel(LoadoutPageContext context)
     {
-        var vm = new LoadoutViewModel(ServiceProvider.GetRequiredService<IWindowManager>(), ServiceProvider, context.LoadoutId);
+        // Default to the user group for now
+        var userGroup = Loadout.Load(_connection.Db, context.LoadoutId).MutableCollections().First().AsLoadoutItemGroup();
```

Comment:
Group should be passed via the context.

Reply:
There's nothing to pass into this yet, I'll add that once I add the UI elements that will populate that value